### ISSUE TITLE
Can't connect to GemStone 32 bits repositories

### ISSUE THREAD

**Issue by gcotelli** [jgfoster/Jade.bak#28]
Trying to connect to 32 bits repositories in Windows gives me always an error like:

**Gcilw66 class does not understand #fileNameSearch
Gcilw65 class does not understand #fileNameSearch**

and after that **JadeSessionManager does not understand #pause**

I'm trying v2.0.6. Find attached the Jade.ERRORS file.

[Jade.ERRORS.zip](https://github.com/jgfoster/Jade/files/291101/Jade.ERRORS.zip)

**Comment by jgfoster** [comment]
Thank you for reporting this. It seems that the 32-bit side of things doesn't get as much testing. I've confirmed now that I can log in to a 32-bit database with Jade 2.0.7.

**Comment by gcotelli** [comment]
@jgfoster , I get a different error now. Find attached an screenshot and Jade.ERRORS file. I'm using Jade 2.0.7 against a GemStone 32 bits 6.6.5 repository.
RPC:
[Jade.ERRORS.zip](https://github.com/jgfoster/Jade/files/293346/Jade.ERRORS.zip)
![jade error](https://cloud.githubusercontent.com/assets/2106744/15709178/b2001e28-27d9-11e6-968e-58433945a642.png)

Linked:
![jade error linked](https://cloud.githubusercontent.com/assets/2106744/15709304/629e39c2-27da-11e6-9aa3-af21084518d7.png)

I can't reopen the issue.

**Comment by jgfoster** [comment]
Sorry; I tested with 6.7 but not 6.6.5 (I'll do that now).

**Comment by jgfoster** [comment]
@gcotelli, the base repository (a new extent0.dbf that is included with the 6.6.5 download) does not include OBGemStonePlatform and does include ClientForwarder; so my login attempt is successful. According to the first error you show above, OBGemStonePlatform exists and ClientForwarder does not exist--the exact opposite of the base system. So, tell me more about your database and what you've done to it. How did OB\* things get loaded? Are you trying to install Seaside code into a 32-bit system?

**Comment by gcotelli** [comment]
Hi @jgfoster,
I've been using a customer repository. It's a 6.6.3 version repository converted to 6.6.5, and it's the same I used on the previous errors. I'm not trying to install anything on it, maybe the OB\* stuff are a consequence of my previous Jade attempts?. Can the ClientForwarder missing be related to something I did wrong during the conversion process?.

Anyway, on Monday I will try with the stock repository on 6.6.3 and 6.6.5 versions, and review the conversion process, and let you know the result.
